annotated-projects/PXD025168/sdrf-plants.tsv

```diff
@@ -0,0 +1,127 @@
+source name	characteristics[organism]	characteristics[organism part]	characteristics[cell type]	characteristics[disease]	characteristics[biological replicate]	technology type	assay name	comment[technical replicate]	comment[data file]	comment[fraction identifier]	comment[label]	comment[instrument]	comment[cleavage agent details]	comment[modification parameters ]	comment[modification parameters ]	comment[modification parameters ]
+KAc_1	Chlamydomonas reinhardtii	whole plant	plant cell	non	1	proteomic profiling by mass spectrometry	KAc_1	1	150311_IF_LeFu_ChlamyII_rep1_Kac_3	13	DiMethyl light	NT=Q Exactive HF	NT=Trypsin	NT=Carbamidomethyl;TA=C;AC=UNIMOD:4;MT=fixed	NT=Oxidation;TA=M;AC=UNIMOD:35;MT=variable	NT=Acetylation;TA=K;AC=UNIMOD:1;MT=variable;PP=Not C-term

```

Comment:
@jayieh We are using a convention for naming the samples like {PXD...}-Sample-{number sample} in your case will be: 

PXD025168-Sample-1
PXD025168-Sample-2
...

Reply:
For the non-available values, instead of `non` please use `not available` or `not applicable` section https://github.com/bigbio/proteomics-metadata-standard/tree/master/sdrf-proteomics#81-sdrf-proteomics-format-rules